src/core/org/apache/jmeter/reporters/ResultCollector.java

```diff
@@ -386,16 +384,14 @@ public void loadExistingFile() {
                             SaveService.loadTestResults(bufferedInputStream,
                                     new ResultCollectorHelper(this, visualizer));
                             parsedOK = true;
-                        } catch (ConversionException e) {
-                            log.warn("Failed to load "+filename+" using XStream. Error was: "+e);
                         } catch (Exception e) {
-                            log.warn("Failed to load "+filename+" using XStream. Error was: "+e);
+                            log.warn("Failed to load " + filename + " using XStream. Error was: " + e);
```

Comment:
Probably, however that would mean changing the rest in this file for consistency. I left it as it was as my intention for this PR was for Java 7 related changes i.e. multi-catch (in this case removing duplicate code and 'fixing' whitespace while there).